android/libraries/rib-base/src/main/java/com/badoo/ribs/core/routing/configuration/Transaction.kt

```diff
@@ -0,0 +1,50 @@
+package com.badoo.ribs.core.routing.configuration
+
+import android.os.Parcelable
+import com.badoo.ribs.core.routing.configuration.ConfigurationCommand.Add
+import com.badoo.ribs.core.routing.configuration.ConfigurationCommand.Remove
+import com.badoo.ribs.core.routing.configuration.action.multi.MultiConfigurationAction
+import com.badoo.ribs.core.routing.configuration.action.multi.SaveInstanceStateAction
+import com.badoo.ribs.core.routing.configuration.action.multi.SleepAction
+import com.badoo.ribs.core.routing.configuration.action.multi.WakeUpAction
+import com.badoo.ribs.core.routing.configuration.feature.TransitionDescriptor
+
+internal sealed class Transaction<C : Parcelable> {
+
+    // TODO move these out to top level maybe
```

Comment:
Do you want to keep this TODO?

Reply:
Yes. TODO doesn't necessarily mean in this branch. Also here it's more like "consider doing so, not sure". We'll have audit, let's decide then.